tests/integration/api/test-validation-function.php

```diff
@@ -540,6 +541,27 @@ public function test_validate_max_will_use_a_custom_exception_message( $value, $
 		validate_max( $value, $max, $message );
 	}
 
+	/**
+	 * @testWith
+	 * [ "Tea", { "minChars": 5 }, "Value must meet the minimum length" ]
+	 * [ "Internationalism", { "maxChars": 10 }, "Value exceeds the maximum length" ]
+	 */
+	public function test_validate_text_field_min_max_will_throw_an_exception_if_string_outside_range( $value, $field, $message ) {
+		$this->expectExceptionMessage( $message );
+		validate_text_min_max( $value, $field );
+	}
+
+	/**
+	 * @testWith
+	 * [ "Seven", { "minChars": 4, "maxChars": 10} ]
+	 * [ "Seven", { } ]
```

Comment:
```suggestion
	 * [ "Seven", { } ]
	 * [ ["Seven", "Eight", "Nine"], { } ]
```

Worth adding a test here for repeating text fields?